packages/core/src/Renderer.ts

```diff
@@ -127,7 +128,7 @@ export class Renderer {
     _renderTextNode(context: RenderingContext): RenderingContext {
         // If the node has a format, render the format nodes first.
         const renderedFormats = [];
-        const firstChar = context.currentVNode;
+        const firstChar = context.currentVNode as CharNode;
```

Comment:
The name `firstChar` is misleading, why not currentCharNode?

Reply:
Because `firstChar` maps to the first character in the text node.